src/collectors/loggly/loggly.py

```diff
@@ -0,0 +1,268 @@
+# coding=utf-8
+
+"""
+Collect numeric value of a field from loggly search results.
+
+This collector will retrieve the values of a configured field from loggly API
+and publish each one on the log's specified timestamp.
+
+The collector will limit the query to a timeframe equal to the collector's
+duration back from now.
+
+#### Dependencies
+
+    * requests
+    * futures
+
+Example config file LogglyCollector.conf
+
+```
+enabled = true
+interval = 60
+
+[ metric ]
+[[ test.metric ]]
+subdomain = example
```

Comment:
You're correct, and this set up is a side effect of the split configuration. I'll move this to the collector level.